jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/console/ConsoleInput.java

```diff
@@ -48,7 +52,7 @@ public void run(String... args) throws Exception {
 		
 		while(scanner.hasNext()) {
 			MDC.put(Constants.MDC_KEY_FILENAME, CONSOLE_FILENAME);
-			TransactionInfo transactionInfo = new TransactionInfo(CONSOLE_FILENAME,"console", LocalDateTime.now());
+			TransactionInfo transactionInfo = new TransactionInfo(CONSOLE_FILENAME, this.accessProperties.getInput().getSender(), LocalDateTime.now());
```

Comment:
Can we get some null pointer exception here?

Reply:
if the sender is a setting, then it should be processed in AccessAutoConfiguration and AccessProperties. When we new TransactionInfo, just get the sender info from Configuration. No need to change each input endpoint. That is my 2 cents.

Reply:
on a second thought, it maybe a not gdod idea

Reply:
not good idea